src/app/router.tsx

```diff
@@ -74,6 +75,10 @@ const router = createHashRouter([
     path: '/my-reports/history',
     element: <ReportsHistory />,
   },
+  {
+    path: '/getstarted',
+    element: <Getstarted />,
+  },
```

Comment:
I noticed that the link for the '/getstarted' path is not properly connected. Please double-check the implementation and make sure the correct URL is assigned to the 'path' property. This will ensure that the page is successfully redirected.